### DISCUSSION TITLE
command line interface

### DISCUSSION THREAD

**Discussion by karubn** [Anjok07/ultimatevocalremovergui#1284] [General]
It would be great if you add a windows command line interface to the current gui version. We expect a windows command line interface in future versions.

**Comment by ghost** [comment]
This has been requested quite often and tbh, I understand if this is very low priority for the dev, due to this being his main source of income. They already replied that its possible by tapping into the py scripts - namely the file seperate.py.

[https://github.com/Anjok07/ultimatevocalremovergui/issues/768#issuecomment-1705714019](https://github.com/Anjok07/ultimatevocalremovergui/issues/768#issuecomment-1705714019)

~edit: I didn't look into this any further either due to having other stuff on my plate, otherwise I would have posted something here :(

**Comment by karubn** [comment]
We respect the developer, but I'm sure she will add the Windows command line interface feature and release it as soon as possible.
